deploy/operator/monitoring-stack-operator-cluster-role.yaml

```diff
@@ -16,14 +16,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
I would guess the operator still needs to watch and list the namespaces.

Reply:
On this I'm actually not sure... since the operator will only have to manage the CRDs no? But maybe I'm missing some context

Reply:
You are right! I blame it on lack of coffee :D